app/api/auth/[...nextauth]/types.ts

```diff
@@ -1,8 +1,8 @@
 export interface UserPayload {
-    id: string
-    profileId: string
-    githubCredentials: {
-      username: string
-      email: string
-    }
-  }
\ No newline at end of file
+  id: string
+  profileId: string
```

Comment:
**nitpick:** `profileId` could be null, because user can exists without it, so: ` profileId: string | null`